pom.xml

```diff
@@ -751,6 +758,30 @@
                     <autoReleaseAfterClose>true</autoReleaseAfterClose>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>${maven.checkstyle.plugin.version}</version>
+                <configuration>
+                    <configLocation>${checkstyle.config.location}</configLocation>
+                    <headerLocation>${checkstyle.header.file}</headerLocation>
+                    <propertyExpansion>samedir=${checkstyle.basedir}</propertyExpansion>
```

Comment:
why we need this?

Reply:
Earlier, I was injecting the header file name using this. But later on, I was able to use default variables to refer to the same. 

Looks like we can go ahead without this particular config.

Reply:
ok, lets remove this.

Reply:
Sure